care/security/authorization/encounter.py

```diff
@@ -10,6 +10,13 @@
 
 
 class EncounterAccess(AuthorizationHandler):
+    def find_roles_on_encounter(self, user, encounter):
```

Comment:
Encounter access can also be given through patient or through location. Ideally get roles on patient should work for encounter as well.